lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1279,7 +1295,13 @@ instance Arbitrary ApiByronWalletBalance where
     arbitrary = genericArbitrary
     shrink = genericShrink
 
-instance Arbitrary ApiWalletMigrationInfo where
+instance Arbitrary ApiWalletMigrationBalance where
+    arbitrary = ApiWalletMigrationBalance
+        <$> reasonablySized arbitrary
+        <*> reasonablySized arbitrary
+    shrink = genericShrink
+
+instance Arbitrary (ApiWalletMigrationInfo n) where
     arbitrary = genericArbitrary
```

Comment:
![Screenshot from 2021-04-29 07-25-30](https://user-images.githubusercontent.com/5680256/116505834-31567600-a8bc-11eb-8a4a-34a0b41d5c86.png)

Be careful with genericArbitrary! This generates data-set that are way too large!

Reply:
> Be careful with genericArbitrary! This generates data-set that are way too large!

Thanks for spotting this! I'll revise the `Arbitrary` instance to generate a smaller data set. 👍🏻